### PR TITLE
Bread icon

### PR DESCRIPTION
**Pull request by jefferykeller**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by larouxn** [comment]
Hey @jefferykeller, sorry, we're no longer accepting new icons at this time. (with very few exceptions) 

https://github.com/activemerchant/payment_icons#-new-icon-contributions-are-not-being-accepted-%EF%B8%8F

**Comment by larouxn** [comment]
👋 Hello @jefferykeller, can we update the icon to look more like other icons, such as [Visa](https://github.com/activemerchant/payment_icons/blob/4830fbc6fefeb5a58d8382428cc6efe9166ffcf9/app/assets/images/payment_icons/visa.svg)?

You can find the guidelines and template here in [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/4830fbc6fefeb5a58d8382428cc6efe9166ffcf9/CONTRIBUTING.md).

Also you can run tests locally with `bundle exec rails test` or view them on GitHub. ([logs](https://github.com/activemerchant/payment_icons/pull/479/checks?check_run_id=3501332706))

**Comment by jefferykeller** [comment]
Thanks @larouxn, I have updated the icon to be centered and have a white background, like the others.

File: app/assets/images/payment_icons/bread.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...383-.09.684-.136a4.3 4.3 0 00.59-.13c.171-.05.308-.116.414-.19zm5.714-4.418v2.547a1.647 1.647 0 00-.628-.715c-.274-.168-.614-.254-1.024-.254-.422 0-.797.094-1.117.285a1.884 1.884 0 00-.754.84 2.45 2.45 0 00-.21.625 4.09 4.09 0 00-.067.75v.125c0 .836.191 1.473.578 1.91.386.442.898.66 1.535.66.445 0 .809-.105 1.094-.32a1.85 1.85 0 00.629-.824l.12 1.04h.946V8.305zm.028 4.191c0 .493-.125.883-.367 1.172-.246.285-.579.43-1 .43a1.19 1.19 0 01-.93-.402c-.242-.266-.363-.668-.363-1.2v-.093c0-.496.117-.883.347-1.16.23-.278.559-.418.989-.418.41 0 .73.144.968.433.239.29.356.672.356 1.152zm0 0" fill="#0f2233"/><path d="M4.953 6.888c-1.343 0-2.305.402-2.305.81 0 .16.106.241.282.288.379.101 3.175.843 3.363.902.223.066 1.344.36 1.344 1.492v-.004S2.219 8.908 1.84 8.826c-.266-.06-.465-.008-.602.128C.906 9.286.5 10.814.5 11.9c0 2.894 1.77 4.988 4.453 4.988 1.344 0 2.305-.402 2.305-.809 0-.16-.106-.242-.281-.289-.38-.1-3.176-.85-3.36-.902-.187-.047-1.347-.36-1.347-1.496l5.796 1.56c.266.057.465.006.602-.13.332-.332.738-1.86.738-2.946 0-2.894-1.77-4.988-4.453-4.988zm-1.336 5.21c-.176-.046-1.347-.366-1.347-1.5l4.027 1.083c.137.035 1.34.371 1.34 1.496 0 0-3.848-1.035-4.02-1.078zm0 0" fill="#5156ea"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/bread.svg:1]: We are missing the borders: [video demonstration](https://screenshot.click/07-49-62cn8-ryfqj.mp4)

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi,

I left some comments.

Please let me know if there are any questions.

**Comment by jefferykeller** [comment]
@tauthomas01, thanks for the detail.  I've combed through all the notes in the contribution guide, and realized that I missed the guidance on the border.  I have added the boarder now, and everything looks right to me as I compare our icon to others.  Please let me know if there is anything else.  Thanks!!

File: app/assets/images/payment_icons/bread.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-bread">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/bread.svg:1]: Can you put the code everything in one line?

You can follow like the [visa icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Review by tauthomas01 — COMMENTED** [review]
Looks good. I just left one very small comment

File: app/assets/images/payment_icons/bread.svg (hunk 2 above)
Comment (jefferykeller) [reply app/assets/images/payment_icons/bread.svg]: Done. Thanks!

**Review by tauthomas01 — APPROVED** [review]

**Review by larouxn — APPROVED** [review]

**Comment by larouxn** [comment]
👋 Just a heads up that we are hoping to have this live across Shopify within the month. Currently waiting on some other icons so we can batch the update. Thank you. 🙇‍♂️

**Comment by larouxn** [comment]
👋  Ran into a few release pipeline issues. Hoping to have this live by end of next week, potentially sooner.

**Pull request merged**